CMakeLists.txt

```diff
@@ -177,7 +177,6 @@ catkin_package(
   CFG_EXTRAS
     rviz-extras.cmake
   INCLUDE_DIRS
-    src
```

Comment:
removing this exported include dir also broke the devel workspace with other dependencies.
RViz does not cleanly split its public headers into `include/` and, e.g., `#include <rviz/frame_manager.h>` will find the system headers and not this one.

Reply:
We still have https://github.com/ros-visualization/rviz/blob/melodic-devel/CMakeLists.txt#L215,
which should define the correct include folder. Which concrete error message to you see? What's the compiler command line?

Reply:
I missed to include EXPORT_HEADER_DIR though. WIP: https://github.com/ros-visualization/rviz/pull/1448

Reply:
I'm building moveit `master` on ROS kinetic and see this error because the includes only find the `/opt/ros/kinetic` header:
```
Errors     << moveit_ros_visualization:make /homeL/pushing/ros/logs/moveit_ros_visualization/build.make.001.log
/homeL/pushing/ros/src/upstream/moveit/moveit_ros/visualization/planning_scene_rviz_plugin/src/planning_scene_display.cpp: In member function ‘virtual planning_scene_monitor::PlanningSceneMonitorPtr moveit_rviz_plugin::PlanningSceneDisplay::createPlanningSceneMonitor()’:
/homeL/pushing/ros/src/upstream/moveit/moveit_ros/visualization/planning_scene_rviz_plugin/src/planning_scene_display.cpp:491:77: error: ‘class rviz::FrameManager’ has no member named ‘getTF2BufferPtr’
   std::shared_ptr<tf2_ros::Buffer> tf_buffer = context_->getFrameManager()->getTF2BufferPtr();
                                                                             ^
```

No problem with the exported `src` include in place.